fineract-provider/src/main/resources/META-INF/spring/jndi.xml

```diff
@@ -29,6 +29,12 @@
     	http://www.springframework.org/schema/jee http://www.springframework.org/schema/jee/spring-jee-3.0.xsd">
 
 	<!-- name here must match TestDataSourceConfiguration -->
-	<jee:jndi-lookup jndi-name="java:comp/env/jdbc/mifosplatform-tenants"
-		id="tenantDataSourceJndi" />
+
+    <bean id="tenantDataSourceJndi" 
+        class="com.zaxxer.hikari.HikariDataSource">
+        <property name="driverClassName" value="org.drizzle.jdbc.DrizzleDriver" />
+        <property name="jdbcUrl" value="jdbc:mysql:thin://localhost:3306/mifosplatform-tenants" />
+        <property name="username" value="root" />
+        <property name="password" value="mysql" />
```

Comment:
we cannot hard-code these parameters here, you must use the same placeholders as in the original `docker/server.xml` - makes sense? (This is also why the `docker-compose` test fails on Travis; the `environment` from `docker-compose.yml` is no longer effective.)